### DISCUSSION TITLE
plant age

### DISCUSSION THREAD

**Discussion by wanyingw193** [NREL/ReEDS-2.0#124] [Q&A: Data]
Hi ReEDS Team, in b_input.gms, there is a parameter plant_age(i,v,r,t), what is the use of this parameter other than to determine retiretech(i,v,r,t)? Thanks!

**Comment by wesleyjcole** [comment]
That is the only thing it is used for.  We use it to make sure we don't retire plants that haven't reached a minimum age threshold.

**Reply by wanyingw193** [reply]
Thank you! I have four more questions.
For the database used to calculate the plant age, I am checking "ReEDS_generator_database_final_EIA-NEMS.csv". 

(1) In row 6423, the "Commercial.Online.Year" for the coal plant is 2016, however, its retirement year is set to 2015. What is the reason for this setting?

(2) Why are some units being entered separately in the database. For example, rows 6391 and 6392, Dry Fork Plant in Wyoming, have the same Plant ID and Unit ID. Why are they entered as two separate rows in "ReEDS_generator_database_final_EIA-NEMS.csv" that can be retired in different years?

(3) If any unit is converted from coal-fired to natural gas combustion during its life cycle, how will it be reflected in the database of EIA-NEMS?

(4) For coal  plants, the input maxage is 70 years, but the lifetime is set to 75 or 65 years. If these two conditions conflict when determining the retirement year, which parameter will be given priority?

**Reply by wesleyjcole** [reply]
(1) In row 6423 I am seeing a coal plant that retires in 2087:

![image](https://user-images.githubusercontent.com/49044852/178300181-a159fe5e-cd04-42f5-83f6-c28a62f5e033.png)

(2) We do not control the separation of plants in this database.  That has already been specified when we get the database from EIA.  EIA does do some separation based on different units within a plant as well as for join ownership of plants by multiple entities.

(3) The "Vintage" column specifies whether a unit has been upgraded or converted, where a value of 6 indicates an upgrade.  We use that along with corresponding retired units and online dates to determine if a plant was converted to natural gas.

(4) The 65/75 values are used when specifying the coal plant retirement dates for anything within the database.  Within the model, maxage for thermal units only applies to new capacity built by the model.

**Reply by wanyingw193** [reply]
(1) I referred the wrong line, sorry, the actual row should be 5101. it has a vintage of 7, so does the 2016 mean it is upgraded to coal-new? Can 65/75 still be used to determine the retirement time after it is assigned as an upgrade technology?
![image](https://user-images.githubusercontent.com/78582696/178312939-81a9089f-e3bd-4957-bff8-fc7a0647a4a2.png)

(2) I misunderstood that the source of the ReEDS database is from the NEMS model. Which EIA database would be the source?

(3) The Vintage column has numbers 1,2,6,7.  I guess, 1 is for normal plants, 6 and 7 are for upgraded plants. What does 2 stand for? Does the vintage number affect the unit-level binning method?

(4) Got it, thank you!

Two more questions.

(a) When assigning bin class to newv tech, are there any rules when defining ivt_defualt and ivt_small? In my understanding, the ivt file refers to the binning method of the new build tech, and the ivt file can be changed by user defined?

(b) Does coal-new refer to supercritical  or subcritical? What are the rules for assigning coal-new?
![image](https://user-images.githubusercontent.com/78582696/178324027-902a200e-e53f-4d20-830a-bfc535d279bd.png)

Thank you!

**Comment by wesleyjcole** [comment]
1) I'm not sure why the online year shows 2016, but it doesn't matter because the retire year overrides it.  This plant was retired in 2015 (https://www.gem.wiki/Silver_Lake_Plant) and will show up in ReEDS as retiring then.

2) This database is the NEMS plant database.  EIA draws from their forms (like EIA-860) in putting it together.

3) 1 means the plant is operating, 2 means the plant is planned

a) The bin assignment in ReEDS (using the "v" set in GAMS) is separate from the plant database.  The ReEDS plant binning occurs in writehintage and bins the plants by heat rate.  Binning of new plants in controlled by the ivt file, which is user defined.

b) The existing coal-new plants take the characteristics as defined in the NEMS database, so they follow the specifications of the actual coal plants.  Any new coal plants built by the model us inputs for a supercritical coal power plant.

**Reply by wanyingw193** [reply]
Thanks!

**Comment by opala** [comment]
All,

My research group is working on an energy flow simulation and visualization
tool for the purpose of virtual water flow analysis. I am running a study
in order to get feedback on the tool and evaluate the flow data that the
simulation generated based on ReEDS data.

We are looking for 5 to 10 individuals who understand energy flow at
regional or national scale. We set up energy flow scenarios for following
cities (representing the Balancing Area region that they reside in).


   - *Seattle *
   - *Houston*
   - *Los Angeles*
   - *Minneapolis*

We show the participants the flows/subflows and energy mix breakdowns that
we simulated on a map and ask them their opinions on each scenario. It
takes around 1.5 hours to complete the study depending on how many regions
you complete. Anyone who has professional experience in energy flows can
sign up by filling out this form <https://forms.gle/DwzG5dMoyrE42mBa8>.
(URL: https://go.ncsu.edu/VirtualWaterStudy )

Once you submit the interest form, we will send you a link to a calendar
where you can pick a time-slot based on your availability. We will award *$40
Amazon gift certificate* to subject matter experts who participate in the
study.

Please see below for the IRB approved invitation letter.



Thanks

Okan Pala, PhD

Center for Geospatial Analytics

North Carolina State University






*Evaluation of the ViWaV (Virtual Water Visualization ) Energy Tracing
Model *

*(eIRB #24458) *

Dr. Okan Pala, ***@***.***, (919)-515-3430



Hello,



You are invited to participate in a research study from the Center for
Geospatial Analytics at NC State University. This study focuses on
evaluating the effectiveness of the Virtual Water Visualization Energy
tracing model, specifically, how effective this model is at creating
projections and its potential to be used as a tool for policymaking and
emergency management.



In order to gain information about how experts view this model, we invite
you to participate in this research study. This study can be completed in
your own time or virtually with a researcher and poses minimal risk. If you
do choose to take it with a researcher, your voice will be recorded,
although identities will be confidential.



Participants will be given data and maps relating to different areas at
different times in the future. The data and the interactive maps were
created using the model. Using your knowledge and expertise in the field,
you will give your opinion on the accuracy of the data and maps. Qualified
participants will* receive a $40 Amazon gift certificate*.



Thank you for your time and consideration. If you are interested in
participating, please sign up by filling out this form
<https://forms.gle/DwzG5dMoyrE42mBa8>. (You can also use following URL :
https://go.ncsu.edu/VirtualWaterStudy )


Please feel free to email Okan Pala with any questions or concerns.



Best,



Okan Pala, PhD

Center for Geospatial Analytics

North Carolina State University

Email: ***@***.***

Jordan Hall 4117

2800 Faucette Dr.

Raleigh, NC 27695 USA



Office Phone: (919)-515 3430

Tiffany Ard

Undergraduate Research Student

Email: ***@***.***
